test/fixtures/local-context.yml

```diff
@@ -60,7 +60,7 @@ build:
 
 changelog:
   -
-    date: 2023-10-10
+    date: 2023-10-10T00:00:00Z
```

Comment:
It is a little annoying to have to specify dates this way. I'm going to look into whether this can be cleaner